scripts/generate-interfaces.js

```diff
@@ -146,13 +146,13 @@ lines.push(
     key: string,
     index?: number,
   }>;
-  declare type TraversalHandler<T> = (BabelNode, TraversalAncestors, T) => void;
-  declare type TraversalHandlers<T> = {
-    enter?: TraversalHandler<T>,
-    exit?: TraversalHandler<T>,
+  declare type TraversalHandler<S: Object> = (BabelNode, TraversalAncestors, S) => void;
```

Comment:
This still seems unnecessary. Limiting to `Object` doesn't seem to have any benefit.